Makefile

```diff
@@ -871,8 +867,7 @@ partialclean::
 
 ocamlc.opt: compilerlibs/ocamlcommon.cmxa compilerlibs/ocamlbytecomp.cmxa \
             $(BYTESTART:.cmo=.cmx)
-	$(CAMLOPT) $(LINKFLAGS) $(OCAML_LDFLAGS) -o $@ \
-	  $^ -cclib "$(BYTECCLIBS)"
+	$(CAMLOPT) $(LINKFLAGS) -o $@ $^ -cclib "$(BYTECCLIBS)"
```

Comment:
I looked at the history of this change (it's quite an old commit where @xavierleroy introduced the use what was then `BYTECCLINKOPTS`) - I think that as things changed further down the line, it simply stayed with an "old" invocation, so removing the use of `LDFLAGS` is OK.